src/screens/TopicPageScreen/index.js

```diff
@@ -329,6 +336,10 @@ const TopicPageScreen = (props) => {
     setIsHeaderHide(false);
   };
 
+  React.useEffect(() => {
+    showHeaderAnimation();
+  }, [coverPath]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This `useEffect` hook triggers the `showHeaderAnimation` function whenever `coverPath` changes. It's a good practice to handle side effects based on dependency changes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->